source/simulator/solver_schemes.cc

```diff
@@ -698,10 +697,8 @@ namespace aspect
                 ||
                 use_picard)
               {
-                pcout << "      Relative nonlinear residual (total Newton system) after nonlinear iteration " << nonlinear_iteration+1
-                      << ": " << test_residual/dcr.initial_residual << ", norm of the rhs: " << test_residual
-                      << ", newton_derivative_scaling_factor: " << newton_handler->parameters.newton_derivative_scaling_factor
-                      << std::endl;
+                pcout << "      Newton system information: Norm of the rhs: " << test_residual
+                      << ", Newton derivative scaling factor: " << newton_handler->parameters.newton_derivative_scaling_factor << std::endl;
```

Comment:
I think with the `Newton system information` in the beginning we can simplify this to:
```suggestion
                      << ", Newton derivative scaling factor: " << newton_handler->parameters.newton_derivative_scaling_factor << std::endl;
```
```suggestion
                      << ", Derivative scaling factor: " << newton_handler->parameters.newton_derivative_scaling_factor << std::endl;
```